addon/webextension/sitehelper.js

```diff
@@ -0,0 +1,41 @@
+/* globals catcher, callBackground */
+/** This is a content script added to all screenshots.firefox.com pages, and allows the site to
+    communicate with the add-on */
+
+window.sitehelper = (function () {
```

Comment:
I created #2573 for this – the refactoring worries me right now, so I'd like to save that work for later.